### PR TITLE
Switching from css to @adobe/css-tools

### LINKED ISSUE #2665: Deprecation warning source-map-resolve
I found some dependency issues related to https://github.com/angular/angular-cli/issues/22748. source-map-resolve seems to be deprecated that is used from the css dependency.

### PR DESCRIPTION
**Pull request by holblin**
**What**:

Switching package `css` for `@adobe/css-tools` , fix #2665 closes #2707 

**Why**:

`css` package is not maintained anymore and contain deprecated dependency

**How**:

**Checklist**:

- [ ] Documentation
- [ ] Unit Tests
- [ ] Code complete
- [ ] Changelog

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4,6 +4,11 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@adobe/css-tools": {
+      "version": "4.0.1",
+      "resolved": "https://artifactory.corp.adobe.com/artifactory/api/npm/npm-adobe-release/@adobe/css-tools/-/css-tools-4.0.1.tgz",
```
Comment (iChenLei) [inline package-lock.json]: @holblin Adobe private npm registry ? We should use `registry.npmjs.org`, can you fix it ?  CI failed [here](https://github.com/stylus/stylus/runs/7701227580?check_suite_focus=true#step:4:13) . Btw, thanks for your contribution.

**Review by iChenLei — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (holblin) [reply package-lock.json]: Yes 👍 I need to redo the installation. I'll do that Monday ;-)

**Comment by holblin** [comment]
@iChenLei I regenerate the `package-lock.json` ;-)

**Review by iChenLei — CHANGES_REQUESTED** [review]
LGTM

**Review by iChenLei — APPROVED** [review]
LGTM

**Comment by holblin** [comment]
I'm curious, what are the next steps before a release/merge?

**Comment by iChenLei** [comment]
@holblin Hi, I only have time to work on open source projects on weekends. So I will check this PR again tomorrow. Maybe I will add more unit tests if needed. and then release 0.59.0. Anyway, thanks for your contribution again.

**Pull request merged**